commercial/app/views/contentapi/items.scala.html

```diff
@@ -44,10 +44,12 @@ <h3 class="ad-slot--paid-for-badge__header">@optSponsorLabel:</h3>
                                             @fragments.items.elements.facia_cards.image(trailPictureContainer, true)
                                         }
                                         <div class="fc-item__content">
-                                            <h2 class="fc-item__title">
-                                                @insertItemMediaIcon(item)
-                                                @item.webTitle
-                                            </h2>
+                                            <div class="fc-item__header">
```

Comment:
__header comes with quite a few styles; I guess we could inject them into __title for this component, but this was the structure used elsewhere - like the fronts - and I suppose I just thought it was best to use it the same way here.